examples/programmatic/scripts/dev.js

```diff
@@ -0,0 +1,9 @@
+const { loadNuxt, build } = require('nuxt')
+
+const main = async () => {
+  const nuxt = await loadNuxt({ for: 'dev' })
+  build(nuxt)
```

Comment:
Should have await here?

Reply:
Not in dev, since it will allow to display our fancy loading screen 😎

Reply:
It might need a good explanation in doc or call await build after listen for consistent usage

Reply:
Well i think no one should use dev programatic usage because it is just a bad DX. It is just to make current usage easier.

Reply:
In create-nuxt-app, we provide this usage 😅

Reply:
Using `await build(nuxt)` could be done after `nuxt.server.listen()` indeed.

Anyway we need to remove the backend framework support in create-nuxt-app to suggest the serverMiddleware usage